feature-pack/src/main/resources/modules/system/layers/base/org/jboss/jts/main/module.xml

```diff
@@ -52,5 +52,11 @@
                 <include path="org.jboss.as.txn.integration"/>
             </imports>
         </module>
+
+        <!-- For the ContextPropagationAsyncHandler used with the context propagation feature pack -->
+        <module name="org.eclipse.microprofile.reactive-streams-operators.api" optional="true" />
+        <module name="org.reactivestreams" optional="true"/>
+        <module name="io.smallrye.reactive.streams-operators" optional="true"/>
+        <module name="io.smallrye.reactive.converters.api" optional="true"/>
```

Comment:
As discussed in chat I will look into creating a convenience module in the feature pack